api/v1/db/oracledb/game-dao.js

```diff
@@ -6,11 +6,9 @@ const { serializeGames } = require('../../serializers/games-serializer');
 const conn = appRoot.require('api/v1/db/oracledb/connection');
 
 /**
- * @summary Merge the raw response from database. Extract card information from every row,
- * and put them into a field called 'tableCard' in the individual merged game object, while other
- * properties in the object remained in the first layer.
- * @function
- * @returns {Promise<Object[]>} Promise object represents a list of games
+ * @summary Return a list of games merged by GAME_ID property.
+ * @param {[Object]} rawGames An array of raw game data returned from SQL database.
+ * @returns {[Object]} Game objects merged.
```

Comment:
Could you add back your longer description in the summary field, add back `@function` and fix the types on `rawGames` and `returns`?